apps/antalmanac/tests/calendarize-helpers.test.ts

```diff
@@ -0,0 +1,227 @@
+                            start: new Date(2018, 0, index, startHour, startMin),
+                            end: new Date(2018, 0, index, endHour, endMin),
+                            finalExam: course.section.finalExam,
+                            isCustomEvent: false as const,
+                        });
+                    }
+                });
+            }
+        }
+    }
+
+    return courseEventsInCalendar;
+}
+
+/**
+ * TODO: Remove this and replace with an array of expected values.
+ */
+export function oldCalendarizeFinals(currentCourses: ScheduleCourse[] = []): CourseEvent[] {
+    const finalsEventsInCalendar: CourseEvent[] = [];
+
+    for (const course of currentCourses) {
+        const finalExam = course.section.finalExam;
+
+        if (
+            finalExam.examStatus == 'SCHEDULED_FINAL' &&
+            finalExam.startTime &&
+            finalExam.endTime &&
+            finalExam.dayOfWeek
+        ) {
+            // TODO: this block is almost the same as in calenarizeCourseEvents. we should refactor to remove the duplicate code.
+
+            const startHour = finalExam.startTime.hour;
+            const startMin = finalExam.startTime.minute;
+            const endHour = finalExam.endTime.hour;
+            const endMin = finalExam.endTime.minute;
+
+            const weekdayInclusion: boolean[] = [
+                finalExam.dayOfWeek.includes('Sat'),
+                finalExam.dayOfWeek.includes('Sun'),
+                finalExam.dayOfWeek.includes('Mon'),
+                finalExam.dayOfWeek.includes('Tue'),
+                finalExam.dayOfWeek.includes('Wed'),
+                finalExam.dayOfWeek.includes('Thu'),
+                finalExam.dayOfWeek.includes('Fri'),
+            ];
+
+            weekdayInclusion.forEach((shouldBeInCal, index) => {
+                if (shouldBeInCal)
+                    finalsEventsInCalendar.push({
+                        color: course.section.color,
+                        term: course.term,
+                        title: course.deptCode + ' ' + course.courseNumber,
+                        courseTitle: course.courseTitle,
+                        bldg: course.section.meetings[0].bldg[0],
+                        instructors: course.section.instructors,
+                        sectionCode: course.section.sectionCode,
+                        sectionType: 'Fin',
+                        start: new Date(2018, 0, index - 1, startHour, startMin),
+                        end: new Date(2018, 0, index - 1, endHour, endMin),
+                        finalExam: course.section.finalExam,
+                        isCustomEvent: false,
+                    });
+            });
+        }
+    }
+
+    return finalsEventsInCalendar;
+}
+
+/**
+ * TODO: Remove this and replace with an array of expected values.
+ */
+export function oldClendarizeCustomEvents(currentCustomEvents: RepeatingCustomEvent[] = []): CustomEvent[] {
```

Comment:
This function name seems like a typo.

Reply:
I'll remove the old functions altogether and replace them with the desired hardcoded values. I kept them around because I was literally checking for compatibility between my re-written version and the one in prod.